From 250839bc2cfeceaa86d48d40b6195a5d0c13a527 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?St=C3=A9phane=20Marchesin?= <marcheu@chromium.org>
Date: Mon, 4 May 2015 14:55:19 -0700
Subject: [PATCH] CHROMIUM: st/mesa: Do not flush front buffer on context flush

Make gallium work again with new chrome.

BUG=none
TEST=compile

Signed-off-by: James Ausmus <james.ausmus@intel.com>
Signed-off-by: Prince Agyeman <prince.agyeman@intel.com>
Signed-off-by: Dhinakaran Pandiyan <dhinakaran.pandiyan@intel.com>
Signed-off-by: Tomasz Figa <tfiga@chromium.org>
Signed-off-by: Gurchetan Singh <gurchetansingh@chromium.org>
---
 src/mesa/state_tracker/st_manager.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/mesa/state_tracker/st_manager.c b/src/mesa/state_tracker/st_manager.c
index 085f54efaa..f396c2a212 100644
--- a/src/mesa/state_tracker/st_manager.c
+++ b/src/mesa/state_tracker/st_manager.c
@@ -507,8 +507,6 @@ st_context_flush(struct st_context_iface *stctxi, unsigned flags,
       st->pipe->screen->fence_reference(st->pipe->screen, fence, NULL);
    }
 
-   if (flags & ST_FLUSH_FRONT)
-      st_manager_flush_frontbuffer(st);
 }
 
 static boolean
-- 
2.12.2

